napari/_qt/widgets/qt_viewer_status_bar.py

```diff
@@ -27,9 +76,51 @@ def __init__(self, parent: '_QtMainWindow') -> None:
         parent._activity_dialog._toggleButton = self._activity_item
         self.addPermanentWidget(self._activity_item)
 
+        self.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Minimum)
+
     def setHelpText(self, text: str) -> None:
         self._help.setText(text)
 
+    def setStatusText(
+        self,
+        text: str = None,
+        layer_base=None,
+        source_type=None,
+        plugin=None,
+        coordinates=None,
+    ) -> None:
+        # The method used to set a single value as the status and not
+        # all the layer information.
+
+        if text:
+            self._status.setText(text)
+        else:
+            self._status.setText(' ')
```

Comment:
```suggestion
            self._status.setText('')
```

Just trying to save as much text as possible 😆